tests/foreman/cli/test_contentview.py

```diff
@@ -161,27 +159,14 @@ def _get_content_view_version_lce_names_set(content_view_id, version_id):
         })['lifecycle-environments']
         return {lce['name'] for lce in lifecycle_environments}
 
-    # pylint: disable=unexpected-keyword-arg
-    def setUp(self):
-        """Tests for content-view via Hammer CLI"""
-
-        super(ContentViewTestCase, self).setUp()
-
-        if ContentViewTestCase.org is None:
-            ContentViewTestCase.org = make_org(cached=True)
-        if ContentViewTestCase.env1 is None:
-            ContentViewTestCase.env1 = make_lifecycle_environment(
-                {u'organization-id': ContentViewTestCase.org['id']})
-        if ContentViewTestCase.env2 is None:
-            ContentViewTestCase.env2 = make_lifecycle_environment(
-                {u'organization-id': ContentViewTestCase.org['id'],
-                 u'prior': ContentViewTestCase.env1['label']})
-        if ContentViewTestCase.product is None:
-            ContentViewTestCase.product = make_product(
-                {u'organization-id': ContentViewTestCase.org['id']},
-                cached=True)
-
-    # pylint: disable=unexpected-keyword-arg
+    @classmethod
+    def setUpClass(cls):
+        """Create an organization, a life cycle environment,
+        and a product."""
```

Comment:
product?

Reply:
Hello, I have added, and pushed: `cls.product` = make_product({u'organization-id': `cls.org['id']})`

Ran tests again as per https://github.com/SatelliteQE/robottelo/pull/6450#issuecomment-437923163